src/components/DateField/index.tsx

```diff
@@ -59,75 +61,84 @@ const DateField = ({
   onFocus = () => {},
   disabled,
   required,
-}: DateFieldProps) => (
-  <Fieldset>
-    <Text as="legend" textWeight={700} marginBottom="xxs">
-      {label} {!!required && ` (required)`}
-    </Text>
-    {hintText && <Text>{hintText}</Text>}
-    <DateTextFieldWrapper>
-      <TextField
-        label="Day"
-        type="text"
-        name={dayName}
-        required={!!required}
-        hideRequiredInLabel
-        maxLength={2}
-        autoComplete="bday-day"
-        pattern="[0-9]*"
-        inputMode="numeric"
-        value={day}
-        onChange={onChange}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        hasError={dayHasError}
-        disabled={disabled}
-      />
-    </DateTextFieldWrapper>
-    <DateTextFieldWrapper>
-      <TextField
-        label="Month"
-        type="text"
-        name={monthName}
-        required={!!required}
-        hideRequiredInLabel
-        maxLength={2}
-        autoComplete="bday-month"
-        pattern="[0-9]*"
-        inputMode="numeric"
-        value={month}
-        onChange={onChange}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        hasError={monthHasError}
-        disabled={disabled}
-      />
-    </DateTextFieldWrapper>
-    <LargeDateTextFieldWrapper>
-      <TextField
-        label="Year"
-        type="text"
-        name={yearName}
-        required={!!required}
-        hideRequiredInLabel
-        maxLength={4}
-        autoComplete="bday-year"
-        pattern="[0-9]*"
-        inputMode="numeric"
-        value={year}
-        onChange={onChange}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        hasError={yearHasError}
-        disabled={disabled}
-      />
-    </LargeDateTextFieldWrapper>
-    {errorMessage && (
-      <ErrorTextWrapper>
-        <ErrorText marginTop="xxs">{errorMessage}</ErrorText>
-      </ErrorTextWrapper>
-    )}
-  </Fieldset>
-);
+}: DateFieldProps) => {
+  const theme = useTheme();
+  const typedTheme = theme as ThemeType;
+  return (
+    <Fieldset>
+      <Text
+        as="legend"
+        textWeight={700}
+        marginBottom="xxs"
+        textFontFamily={typedTheme.typography.fontFamilyLabel}
+      >
+        {label} {!!required && ` (required)`}
+      </Text>
```

Comment:
`fontFamilyLabel` of semi bold for CRUK theme looks a bit too heavy. Do you know why this design has been changed?

![image](https://github.com/CRUKorg/cruk-react-components/assets/102517887/d328d3ac-c466-47ef-b71d-7a241be97fbd)

Reply:
The font has changed managing the font weights is a bit of mare becuause the values in the figma don't match reality so this is a bit more of an art than science I'll see if I can find something that claims to be 400 if base is now 300

Reply:
![Screenshot 2023-10-20 at 11 55 01](https://github.com/CRUKorg/cruk-react-components/assets/5347198/e9eee41a-0761-498f-983c-6f547a17990e)
 my browser doesn't show it as heavy which is really interesting

Reply:
I have overhauled @font-face and font-weights probably worth another look